### DISCUSSION TITLE
Get values of another list by lookup

### DISCUSSION THREAD

**Discussion by HeKr09** [microsoft-search/pnp-modern-search#3255] [Q&A]
I have got a list that I want to show within the search results. This works fine at all.
Nevertheless there is a lookup column within the list that refers to another list.

**List RESULTS**
- Title:Text
- Answer:Text
- Additional:Text
- Keyword:Lookup

**List KEYWORDS**
- Title:Text
- Supplementary:Text
- _ChildKeywords:Lookup_

The "ChildKeywords" column is a multi lookup on the list keywords.

What I want to do is to show in the search results is the supplementary text according to the result item's keyword

**Comment by HeKr09** [comment]
Is the pnp-lookup-list-expander an possible option? But this is v3 and I do not found any documentation within v4.

**Reply by wobba** [reply]
If someone adds support for it, sure.

**Reply by HeKr09** [reply]
What do you mean @wobba? Is it implemented? Did it work in v3 already or not? Were there issues already in v3?
Is there any other alternative?

**Reply by wobba** [reply]
It's not implemented. A developer has to create this and contribute it for the community to use.

**Reply by HeKr09** [reply]
Is there maybe a chance to use a second search result webpart that uses the hovered item from the first search result to search itself? Maybe a stupid idea but something like a "customers who bought X are also interesseted in A, B, C".

**Reply by kasperbolarsen** [reply]
Master-Detail relationships can use the connected results web parts featuring as per https://microsoft-search.github.io/pnp-modern-search/usage/search-results/connections/

**Reply by HeKr09** [reply]
@kasperbolarsen: I'll try this.

**Reply by HeKr09** [reply]
@kasperbolarsen: This seems to work but do you know why the selection of the source web part is unselected if you click on the next page of the target web part?

**Reply by kasperbolarsen** [reply]
It shouldn't. Are you on version 4.9.x?

**Comment by HeKr09** [comment]
@kasperbolarsen: I am working with 4.9.0

**Reply by HeKr09** [reply]
@kasperbolarsen: Any idea?

**Reply by kasperbolarsen** [reply]
Unfortunately not. I might have to see it to figure out what might be the problem.  
I am running PnP Modern Search Office Hours every other Tuesday where people can sign up for a 15 min session via Teams, looking at a specific search related issue. If this is of interest you can sign up here, aka.ms/pnp/modernsearchofficehours
